tests/misc/multilisteneer.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) The One True Way 2021. Apache License 2.0. The authors accept no liability, 0 nada for the use of this software.  It is offered "As IS"  Have fun with it!!
+ */
+
+package main
+
+import (
+	"fmt"
+	"github.com/nats-io/nats.go"
+	log "github.com/sirupsen/logrus"
+	"time"
+)
+
+func main() {
+	nc1, err := nats.Connect("localhost:30220", nats.ClosedHandler(func(_ *nats.Conn) {
+		fmt.Printf("Connection close  \n")
+	}),
```

Comment:
FYI: You could use the `testing` package to make this an actual test, and run it with `go test tests/misc/multilisteneer.go`. That way, you don't need to make a main func and package, and it could be automated at some point.